scss/_patterns_aside.scss

```diff
@@ -0,0 +1,58 @@
+@import 'settings';
+
+@mixin vf-p-aside {
+  .p-aside {
+    align-self: flex-start;
+    border-top: 1px solid $color-mid-light;
+    flex: 0 0 16rem;
```

Comment:
Shouldn't we use the grid to define the width of this pattern? All patterns should be fluid within its parent container.

Reply:
As lifted from vanilla-docs-theme, this is a pattern that is positioned outside of the page flow so it will be fixed to the top of the viewport as a user scrolls which is why I presume it is not in the grid to start with?

Reply:
I think this should be removed as the docs theme was created quickly and without the full process. I think navigation should be fluid with its container.